features/admin.branding.v1/components/preview/branding-preference-preview.tsx

```diff
@@ -409,6 +409,12 @@ export const BrandingPreferencePreview: FunctionComponent<BrandingPreferencePrev
                             >
                                 { t("common:learnMore") }
                             </DocumentationLink>
+                            <a
+                                href="https://is.docs.wso2.com/en/latest/guides/branding/add-custom-layouts/" 
```

Comment:
Let's not hardcode the URL here. Please refer to other places where we have dynamically obtained the documentation link and follow the same approach here.